From 30de542827be41fff3c7fee5e6b2127c03b70f8e Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 28 Apr 2021 21:17:50 +0300
Subject: [PATCH] [SW-0] lkd: remove redundant condition for clr_str

The CLR_STS command handling in run fast handler is not "fall through",
so remove the condition for checking the print out string setting.

Change-Id: Ib112bff84915f8744f2147937ca428a0c3e5c897
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_lkd.c b/preboot/module_lkd.c
index a2e37f3..0a9b663 100644
--- a/preboot/module_lkd.c
+++ b/preboot/module_lkd.c
@@ -275,7 +275,7 @@
 		module_lkd_run_fast_noop();
 		break;
 	case COMMS_CLR_STS:
-		cmd_str = cmd_str ?: "clr_sts";
+		cmd_str = "clr_sts";
 		/* Clear the status */
 		module_lkd_send_status(COMMS_STS_NOOP, COMMS_SRAM, 0);
 		break;
